trunk/SUAVE/Components/Wings/Segment.py

```diff
@@ -42,7 +42,7 @@ def __defaults__(self):
         self.dihedral_outboard     = 0.0
         self.sweeps                = Data()
         self.sweeps.quarter_chord  = 0.0
-        self.sweeps.leading_edge   = 0.0
+        self.sweeps.leading_edge   = None
```

Comment:
why None? Not that it's wrong

Reply:
This change was for AVL, if both arguments have numerical values, there was no way of determining which one to use. A better approach was to set one to None, and if defined, use that one.